src/main/java/uk/nhs/hee/tis/trainee/forms/dto/FormRPartBDto.java

```diff
@@ -72,4 +72,5 @@ public class FormRPartBDto {
   private CovidDeclarationDto covidDeclarationDto;
   private Boolean haveCurrentUnresolvedDeclarations;
   private Boolean havePreviousUnresolvedDeclarations;
+  private Number arcpYear;
```

Comment:
I don't think I've ever seen the `Number` abstract actually used 😅 
Maybe use [Year](https://docs.oracle.com/javase/8/docs/api/java/time/Year.html) instead?